### PR TITLE
Pavlo

### PR DESCRIPTION
**Pull request by PavloTarnovetskyi**
**What is this change?**
In main.tf changed the protocol to '-1', deleted deploying script, and wrote deploying task in playbook.yml

**Why is this important?**
used better deploying practice

**How has this been tested?**
manually from localhost

**How can this be rolled back?**

Rollback can be performed by reverting these changes in source control and redeploying via standard deployment mechanisms. This change is safe to roll back.

**Are there any security risks introduced by this change?**

### CODE REVIEW

File: ansible/playbook.yml [hunk 1]
```diff
@@ -10,45 +10,18 @@
...
-    become: yes
-
-  - name: Install docker
-    apt:
-      name: docker.io
-    become: yes
-
-  - name: Start Docker
+  - name: Install docker and run flaskapp.
     shell: |
```
Comment (antoniakstas) [inline ansible/playbook.yml]: this needs to be updated to use corresponding ansible modules. For example for [apt](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html) for [systemctl](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html) etc..

**Review by antoniakstas — CHANGES_REQUESTED** [review]

File: ansible/playbook.yml (hunk 1 above)
Comment (PavloTarnovetskyi) [reply ansible/playbook.yml]: ok

Comment (nos373737) [reply ansible/playbook.yml]: @antoniakstas could you please review?

File: ansible/playbook.yml [hunk 2]
```diff
@@ -10,45 +10,33 @@
...
-  - name: Add remote user to docker group
-    user:
-      name: "ubuntu"
-      group: "docker"
-    append: yes
-
   - name: Run image
-    shell: docker run --name flaskapp_jenkins -p 5000:5000 -d pavlotarnovetskyi/flaskapp_jenkins
+    shell: docker run --name flaskapp_jenkins -d -p 5000:5000  -d pavlotarnovetskyi/flaskapp_jenkins
     become: yes
```
Comment (antoniakstas) [inline ansible/playbook.yml]: there are also modules for [systemctl](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html) and [docker container](https://docs.ansible.com/ansible/latest/collections/community/docker/docker_container_module.html#ansible-collections-community-docker-docker-container-module)

Comment (nos373737) [reply ansible/playbook.yml]: As I understand you need to get out of using systemctl via ansible modules and docker commands the same

**Review by antoniakstas — COMMENTED** [review]
what is the file .DS_Store? you've added it to gitignore, but its still here

File: DaS/flaskapp_diagrama.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    
+    
+    
+   
+    
+   
+
+
+
```
Comment (antoniakstas) [inline DaS/flaskapp_diagrama.py]:
```suggestion

```

**Review by antoniakstas — APPROVED** [review]
lgtm

**Pull request merged**